test/e2e_node/util.go

```diff
@@ -57,8 +57,11 @@ var startServices = flag.Bool("start-services", true, "If true, start local node
 var stopServices = flag.Bool("stop-services", true, "If true, stop local node services after running tests")
 var busyboxImage = "busybox"
 
-// Kubelet internal cgroup name for node allocatable cgroup.
-const defaultNodeAllocatableCgroup = "kubepods"
+const (
+	// Kubelet internal cgroup name for node allocatable cgroup.
+	defaultNodeAllocatableCgroup = "kubepods"
+	kubeletConfigKey             = "kubelet"
```

Comment:
I'd inline this rather than make a constant

Reply:
done